CHANGELOG.md

```diff
@@ -2,6 +2,7 @@ CHANGELOG for Sulu
 ==================
 
 * dev-develop
+    * ENHANCEMENT #2130 [All]                 Add support for newer symfony distributions with `bin/` directory
```

Comment:
Should be `[MediaBundle]` instead of `[All]`, because your fix is located there :-)